packages/backend-plugin-api/src/services/definitions/coreServices.ts

```diff
@@ -14,17 +14,165 @@
-export { tokenManagerServiceRef as tokenManager } from './tokenManagerServiceRef';
-export { permissionsServiceRef as permissions } from './permissionsServiceRef';
-export { schedulerServiceRef as scheduler } from './schedulerServiceRef';
-export { rootLifecycleServiceRef as rootLifecycle } from './rootLifecycleServiceRef';
-export { rootLoggerServiceRef as rootLogger } from './rootLoggerServiceRef';
-export { pluginMetadataServiceRef as pluginMetadata } from './pluginMetadataServiceRef';
-export { lifecycleServiceRef as lifecycle } from './lifecycleServiceRef';
+import { createServiceRef } from '../system';
+
+/**
+ * All core services references
+ *
+ * @public
+ */
+export namespace coreServices {
+  /**
+   * The service reference for the plugin scoped {@link CacheService}.
+   *
+   * @public
+   */
+  export const cache = createServiceRef<import('./CacheService').CacheService>({
+    id: 'core.logger',
+  });
+
+  /**
+   * The service reference for the root scoped {@link ConfigService}.
+   *
+   * @public
+   */
+  export const config = createServiceRef<
+    import('./ConfigService').ConfigService
+  >({
+    id: 'core.root.config',
+    scope: 'root',
+  });
+
+  /**
+   * The service reference for the plugin scoped {@link DatabaseService}.
+   *
+   * @public
+   */
+  export const database = createServiceRef<
+    import('./DatabaseService').DatabaseService
+  >({
+    id: 'core.database',
+  });
+
+  /**
+   * The service reference for the plugin scoped {@link DiscoveryService}.
+   *
+   * @public
+   */
+  export const discovery = createServiceRef<
+    import('./DiscoveryService').DiscoveryService
+  >({
+    id: 'core.discovery',
+  });
+
+  /**
+   * The service reference for the plugin scoped {@link HttpRouterService}.
+   *
+   * @public
+   */
+  export const httpRouter = createServiceRef<
+    import('./HttpRouterService').HttpRouterService
+  >({ id: 'core.httpRouter' });
+
+  /**
+   * The service reference for the plugin scoped {@link LifecycleService}.
+   *
+   * @public
+   */
+  export const lifecycle = createServiceRef<
+    import('./LifecycleService').LifecycleService
+  >({
+    id: 'core.lifecycle',
+    scope: 'plugin',
+  });
+
+  /**
+   * The service reference for the plugin scoped {@link LoggerService}.
+   *
+   * @public
+   */
+  export const logger = createServiceRef<
+    import('./LoggerService').LoggerService
+  >({
+    id: 'core.logger',
+  });
+
+  /**
+   * The service reference for the plugin scoped {@link PermissionsService}.
+   *
+   * @public
+   */
+  export const permissions = createServiceRef<
+    import('./PermissionsService').PermissionsService
+  >({
+    id: 'core.permissions',
+  });
+
+  /**
+   * The service reference for the plugin scoped {@link PluginMetadataService}.
+   *
+   * @public
+   */
+  export const pluginMetadata = createServiceRef<
+    import('./PluginMetadataService').PluginMetadataService
+  >({
+    id: 'core.plugin-metadata',
```

Comment:
The others are camel cased - just noting

Reply:
Yep wanna fix all of that in a followup